ttnn/examples/usage/using_dropout.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+
+import ttnn
+import torch
+import numpy as np
+
+
+def get_tensor_ratio(t, out):
+    t_sum = t.sum().item()
+    out_sum = out.sum().item()
+    return 1.0 - out_sum / t_sum
+
+
+with ttnn.manage_device(device_id=0) as device:
+    device.enable_program_cache()
+    device.enable_async(True)
+    t = torch.ones(
+        (
+            32,
+            1,
+            32,
+            32,
+        )
+    )
+    t_tt = ttnn.from_torch(t, device=device, dtype=ttnn.bfloat16, layout=ttnn.TILE_LAYOUT)
+
+    ratios = []
+    s = 124
+    prob = 0.2
+    for _ in range(100):
+        output = ttnn.dropout(t_tt, seed=s, probability=prob, scale=1.0)
```

Comment:
Discussed offline. My opinion that we need to change scale to 1/prob and change output = dropout(t) (without * (1 - prob)). Feel free to choose any option.